src/makeCalculator.js

```diff
@@ -37,7 +37,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(firstNum, secondNum) {
+      return firstNum + secondNum;
+    },
+
+    subtract(firstNum, secondNum) {
+      return firstNum - secondNum;
+    },
+
+    divide(firstNum, secondNum) {
+      if (secondNum !== 0) {
+        return firstNum / secondNum;
+      }
+
+      return this;
```

Comment:
Returning 'this' when division by zero is not appropriate. Should throw an error instead.